google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineFacetConfiguration.java

```diff
@@ -50,17 +58,33 @@ public void readExternal(Element element) throws InvalidDataException {
   public void writeExternal(Element element) throws WriteExternalException {
   }
 
+  public Set<AppEngineStandardMavenLibrary> getLibraries(@NotNull Project project) {
```

Comment:
This method could use some Javadoc. Seems like its converting libraries from the platform representation to ours?